database/migrations/2022_03_22_114700_create_consumables_stock.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateConsumablesStock extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('consumables_stock', function (Blueprint $table) {
```

Comment:
I feel like the way this table is used is that it really is a measure of `Replenishments` so maybe we should rename it to that?

Reply:
noted

Reply:
table renamed to consumables_replenishments now